docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -249,6 +249,15 @@ Accessors
 
     Returns the great-circle distance in meters between two SphericalGeography points.
 
+.. function:: geometry_nearest_points(Geometry, Geometry) -> row(Point, Point)
```

Comment:
This should go in the "Operations" section above. Also, make sure to place in the correct alphabetical position.

Reply:
Done.